tests/test_mixins.py

```diff
@@ -1478,13 +1478,28 @@ def test_write_to_postgres_is_called_for_uploading_a_table_with_columns_with_env
         ).get(source_key="WRITE_TO_PG_PARQUET")
 
         # When
-        WritePostgresIO(source_config=postgres_cloud_config, model=ERModel).write(df)
+        WritePostgresIO(source_config=postgres_cloud_config).write(df)
 
         # Then
         mock__write_to_postgres.assert_called()
 
     @pytest.mark.unit
-    def test_read_from_postgres_raises_keyerror_if_no_model_or_no_sql_query_in_options(self):
+    @patch.object(WithPostgres, "_read_from_postgres")
+    def test_read_from_postgres_if_no_sql_query_in_options(self, mock__read_from_postgres, test_df):
```

Comment:
Rename to "test_read_from_postgres_by_implicitly_generating_datamodel_from_schema"